amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2495,7 +2497,194 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  ##retention-schedule
+  '/record-categories/{recordCategoryId}/retention-schedules':
+    post:
+      tags:
+        - retention-schedule
+      summary: Create a retention schedule
+      description: |
+        Create a retention schedule.        
+        For example, using the following JSON body will create a retention schedule:
+        ```JSON
+          {
+              "authority":"Retention Authority",
+              "instructions":"Retention Instructions",
+              "isRecordLevel":false
+          }
+        ```
+      operationId: createRetentionSchedule
+      parameters:
+        - $ref: '#/parameters/recordCategoryIdParam'
+        - in: body
+          name: nodeBodyCreate
+          description: |
+            The node information to create.
+          schema:
+            $ref: '#/definitions/RetentionNodeBodyCreate'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/RetentionScheduleResponse'
+        '400':
+          description: |
+            Invalid parameter: recordCategoryId is not a valid format
+        '401':
+          description: Authentication failed
+        '403':
+          description: Current user does not have permission to create retention schedule
+        '404':
+          description: recordCategoryId does not exist
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    get:
+      tags:
+        - retention-schedule
+      summary: Get a list of retention schedules
+      description: |
+        Get a list of retention schedules.
```

Comment:
It would be worth mentioning that a category can have at most a single schedule attached to it.